### DISCUSSION TITLE
Help: Select Audio Track (to hear in the editor)

### DISCUSSION THREAD

**Discussion by onkoe** [mifi/lossless-cut#2060] [Support]
I can't seem to find the button to change the audio track you're currently hearing in the editor.

Opening the tracks tab only allows me to include/exclude tracks from export - not from my ears. 

Does anyone know how to change tracks? I saw a PR that implements it but still fell short of finding it. Thank you! :)

**Comment by mifi** [comment]
it's in the bottom right of the player. there's a subtitle icon 
<img width="316" alt="Screenshot 2024-08-05 at 23 59 25" src="https://github.com/user-attachments/assets/167a7961-b6ea-4278-a08c-c7eeb7e1349a">

if you don't see it, make sure you're on the latest version.

**Reply by lovelyummy** [reply]
i have 2 audio tracks mic and just game sound , may i hear both of this tracks together? i would like to hear 2 tracks

**Reply by mifi** [reply]
Not possible, see #256
